Add links for feedback + editing docs

henrikvtcodes opened the pull request:
Addresses #12  
Please let me know if the style isn't consistent with the rest of the docs - I did my best there lol. Also - since the edit + feedback links are in the same container as the ToC, they disappear when the page shrinks. Let me know if this is a problem.

henrikvtcodes commented:
I just realized the edit link doesn't actually work - will fix

henrikvtcodes commented:
It has been fixed

IHIutch commented on apps/docs/app/getting-started/[[...slug]]/page.tsx:
```diff
@@ -69,14 +70,18 @@ export default async function GettingStartedPage({
     notFound()
   }
-
+  post._raw.sourceFilePath
```
Remove

IHIutch replied:
Remove

IHIutch commented on apps/docs/components/edit-feedback.tsx:
```diff
@@ -0,0 +1,37 @@
+  contentPath: string
+}
+
+export default function EditFeedbackLinks({ post, contentPath }: Props) {
```
Let's just pass the post title as a prop, rather than the whole post, since that's all we need here

IHIutch commented on apps/docs/components/edit-feedback.tsx:
```diff
@@ -0,0 +1,37 @@
+  pageFeedbackUrl.searchParams.set('labels', 'documentation')
+
+  return (
+    <span className="flex flex-col space-y-2 pt-6">
```
Couple notes here:
- Let's use a list for the links here. Semantically, I'm not entirely sure it matters, but that's what we do for the ToC and I'd rather opt for consistency if nothing else. (In general, I'd avoid converting spans to non-inline elements. This has the potential to create invalid HTML if you wanted to make your links display: block, for example)
- Let's also use the link style/spacing from the ToC links for consistency

IHIutch reviewed (changes requested):
This is great, a couple notes. Thanks!

henrikvtcodes commented:
Cleaned up the code as per your suggestions. As for the style and spacing of the links, I did my best to imitate the spacing. The text styles for color, darkmode, hover, etc. were copy and pasted from the ToC. I made it a list, removed the flexbox, added `py-1` on the links themselves and `mt-3` on the `<ul>` to imitate the spacing between On This Page and the ToC links.

IHIutch commented on apps/docs/components/edit-feedback.tsx:
```diff
@@ -0,0 +1,36 @@
+    <ul className="mt-3">
+      <li className="py-1">
+        <a
+          className="text-xs text-slate-600 hover:text-slate-900 dark:text-slate-400 dark:hover:text-white"
```
Double-check the link styles from the ToC and these. I'd like them to be as consistent as possible. I don't think you'll be able to copy/paste exactly, because the ToC links have an "active" state, but you should be able to copy over the non-active styles. If might even be more helpful to copy from your Inspector because that would be the outputted classes that the DOM sees, without the `cx` function.

If you're able to do that, I think you'll be able to drop the `py-1` on the list items.

henrikvtcodes replied:
Just to clarify what I am looking for: do you want the edit/feedback links to look exactly the same (incl. text size, spacing, colors, hover) or is there a specific inconsistency?

IHIutch replied:
Good question. These links the same as the ToC links

IHIutch reviewed (commented):
No worries, appreciate your work here! One minor note and I think we're ready to go. Thank you!

henrikvtcodes commented:
I copied the styles from the ToC links! It looks like what you requested. @IHIutch

IHIutch commented:
Closes #12
